[IMP] Administration: Add client database info (#12256)
odoo/documentation: content/administration/odoo_accounts.rst

Odoo.com accounts
=================

This document describes how to edit and manage an Odoo.com account, including adding a client
database, deleting an account, resetting a password, and enabling two-factor authentication.

Grant users access to database
==============================

A database can be linked to an Odoo.com account that was created by a different Odoo.com account.

> This allows the database to appear in the **My Databases** page of the account. To do so, the

Reviewer: We use italics when referencing a page we're not directly looking at

Odoo.com accounts
=================

> This document describes how to edit and manage an Odoo.com account. This includes common processes,

Reviewer 2: ```suggestion
This document describes how to edit and manage an Odoo.com account. This includes common processes
```

> such as adding a client database to an Odoo.com account, how to delete an Odoo.com account, how to

Reviewer 2: ```suggestion
such as how to add a client database to an Odoo.com account, how to delete an Odoo.com account, how to
```

Reviewer 2: Edited for parallel listing syntax

> reset an Odoo.com account password, and how to add two-factor authentication.

Reviewer 2: ```suggestion
reset an account password, and how to add two-factor authentication.
```

Add a client database to an Odoo.com account

> ==============================================

Reviewer 2: ```suggestion
============================================
```

A database can be linked to an Odoo.com account that was created by a different Odoo.com account.
This allows the database to appear in the **My Databases** page of the account. To do so, the
desired Odoo.com account needs to be added to the database as a user.

First, sign in to the Odoo.com account that created the desired database. In the main Odoo dashboard

> of the database, navigate to :menuselection:`**Settings** --> Manage Users`. Click the

Reviewer 2: ```suggestion
of the database, navigate to :menuselection:`Settings --> Manage Users`. Click the
```

:guilabel:`New` button, and enter a name in the :guilabel:`Name` field for this user. Next, enter
the email address used to register the desired Odoo.com account in the :guilabel:`Email` field.
Navigate back to :guilabel:`Settings` using the breadcrumbs, and a pop-up window titled
:guilabel:`Users Invitations` appears, with a warning that adding additional users will increase the
amount of the subscription.

.. image:: odoo_accounts/users-invitations.png
   :alt: The `Users Invitations` pop-up window with the options to `Confirm` or `Cancel`.

Click :guilabel:`Confirm`, and the :guilabel:`Users` list appears. An invitation is automatically

> sent to the email address. Once confirmed, the database now appears in the **My Databases** page of

Reviewer 2: ```suggestion
sent to the email address. Once confirmed, the desired database now appears in the **My Databases** page of
```

> the account.

Reviewer 2: ```suggestion
the target Odoo.com account.
```

Reviewer 2: Optional since it could be considered redundant, but I would consider being very explicit about which Odoo.com account the database appears in after the steps

> .. tip::

Reviewer 2: Optional suggestion: could be useful to add a screenshot of the recommended settings

   To provide full access to the database for this user, set the :guilabel:`Dashboard` field under
   the :guilabel:`Productivity` header to :guilabel:`Admin`. Next, set the
   :guilabel:`Administration` field under the :guilabel:`Administration` header to
   :guilabel:`Settings`.

.. example::

>    Deco Addict wishes to have the database 'Stealthy Wood' appear in their 'My Databases'

Reviewer 2: ```suggestion
   Deco Addict wishes to have the database **Stealthy Wood** appear in their **My Databases**
```

>    page, but 'Stealthy Wood' was created by Mitchell Admin. To add the account, a user with

Reviewer 2: ```suggestion
   page, but **Stealthy Wood** was created by a different user, Mitchell Admin. To add the account, a user with
```

>    administrator access of 'Stealthy Wood' adds Deco Addict as a user of 'Stealthy Wood'.

Reviewer 2: ```suggestion
   administrator access of the **Stealthy Wood** database adds Deco Addict as a user of **Stealthy Wood**.
```

>    Now when Deco Addict visits their 'My Databases' page, 'Stealthy Wood' appears.

Reviewer 2: ```suggestion
   Now when Deco Addict visits their **My Databases** page, the **Stealthy Wood** database appears, and they have user access.
```

Odoo.com accounts
=================

This document describes how to edit and manage an Odoo.com account. This includes common processes
such as how to add a client database to an Odoo.com account, how to delete an Odoo.com account, how
to reset an account password, and how to add two-factor authentication.

> Add a client database to an Odoo.com account

Reviewer: After reading through the section, I think the title might not fully match the instructions provided here. From the title, it seems like we're connecting two databases, but it looks like the instructions are actually about inviting a user to another database by email.

How about we go with something like: 'Add user to Odoo.com' or 'Grant users access to database' instead?
